Samples/word-add-in-get-set-edit-openxml/README.md

```diff
@@ -0,0 +1,113 @@
+
+**Note**
+
+The sample uses Office Open XML (the ooxml coercion type) rather than HTML or plain text because OOXML coercion enables you to interact with virtually every type of content a user can insert in their document, such as in the examples you see in Figures 1a - 1b.
+
+Figure 2. Task pane appearance after using the Get… button to extract Office Open XML for selected content
+
+![Screenshot of the add-ins task pane opened in Word](/description/add-in-screenshot.png)
+
+When you select content and choose the 'Get…' button, the add-in uses the JavaScript **getSelectedDataAsync** method to generate a complete Office Open XML document package that includes the selected document content, and places it in the text area of the task pane (as shown in Figure 2).You can use the text area in that task pane as a scratch pad. Copy the Office Open XML markup you retrieve into an XML file, edit it to include just the information you need, and paste it back into the text area of the task pane to test your edited markup.
+
+After you paste or edit the content in the text area of the task pane, you can click in a blank area of the document and then choose the **Insert…** button on the task pane to test the integrity of your markup. That button uses the JavaScript **setSelectedDataAsync** method to insert the contents of the task pane text area as rich content in Word, using the ooxml coercion type.
+
+## Key components of the sample
+
+The sample contains:
+
+- The WD_OpenXML_js project, which contains:
+    - The WD_OpenXML_js.xml manifest file
+    - The SampleDoc.docx document, which is prepopulated with various types of rich content
+- The WD_OpenXML_js Web project, which contains multiple template files. However, the two files that have been developed as part of this sample solution include:
+    - WD_OpenXML_js.html (in the Pages folder). This contains the HTML user interface that is displayed in the task pane. It consists of two HTML buttons that extract and insert Office Open XML, a DIV where status messages will be written, a textarea HTML control that is used to show you Office Open XML markup, and instructional text
+    - WD_OpenXML_js.js (in the Scripts folder). This script file contains code that runs when the add-in is loaded. The code checks for which APIs are supported by the host application. If the host is 2013, the application uses the common Office.js API. If the host is Word 2016 or later, the add-in uses the Word Javascript API. This startup wires up the **onclick** event handlers for the two buttons in WD_OpenXML_js.html. One of these buttons retrieves the selected document content as Office Open XML, and the other button inserts content into the document via OOXML coercion, using the contents of the text area in the task pane. All other files are automatically provided by the Visual Studio project template for an Office add-in, and they have not been modified in the development of this sample app
```

Comment:
```suggestion
    - **WD_OpenXML_js.js** (in the Scripts folder). This script file contains code that runs when the add-in is loaded. The code checks for the APIs that are supported by the host application. If the host is 2013, the application uses the common Office.js API. If the host is Word 2016 or later, the add-in uses the Word JavaScript API. This startup installs the **onclick** event handlers for the two buttons in WD_OpenXML_js.html. One of these buttons retrieves the selected document content as Office Open XML, and the other button inserts content into the document via OOXML coercion, using the contents of the text area in the task pane. All other files are automatically provided by the Visual Studio project template for an Office Add-in, and they have not been modified in the development of this sample app.
```
"wires up" is vernacular, not good for loc. Choose the appropriate word like "install" or "connect" to convey meaning.